pkg/logwrite/logwrite.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+const (
+	logDump byte = iota
+	logFollow
+	logDumpFollow
+)
+
+// LogMessage is a single message sent to memlogd
```

Comment:
This comment seems imprecise. We *receive* a comma/semicolon separate string from memlogd and turn it into this struct.